x/oracle/keeper/upgrade.go

```diff
@@ -58,3 +58,74 @@ func (k Keeper) ApplyUpgrade(ctx sdk.Context, info *types.OracleUpgradeInfo) err
 	ctx.Logger().Info("Oracle upgrade was successful.", fmt.Sprintf("uniqueID: %s, height: %v", info.UniqueId, info.Height))
 	return nil
 }
+
+func (k Keeper) UpgradeOracle(ctx sdk.Context, msg *types.MsgUpgradeOracle) error {
+	oracleUpgrade := types.NewUpgradeOracle(msg)
+
+	if err := oracleUpgrade.ValidateBasic(); err != nil {
+		return err
+	}
```

Comment:
We don't need this validation. This basic validation probably would be done when validate each message (in this case `MsgUpgradeOracle`)
Accordingly, there is no need to have `OracleUpgrade`'s `ValidateBasic()` in `types/oracle.go`.